expirationd.lua

```diff
@@ -25,6 +25,7 @@ end
 local stash_names = {
```

Comment:
So why we don't restart tasks after reload? (Again, I'm not experienced in expirationd.)

Reply:
This is debatable, but it doesn't look like we should do it here:

1. If an user manually calls expirationd.start() from his cartridge role - nothing stops him from just doing it again after hot-reload.
2. From our own cartridge role we can write the logic.

At the expirationd-utility level is difficult to provide the correct behavior for all cases (for example, after hot-reload a config changes and user does not need the task anymore, or a space was deleted, or an index). For my taste, it's better to implement such logic on a higher level - cartridge-role-level.

Therefore, I provide only basic support of cartridge hot-reload here.